### PR TITLE
Version 0 0 1

### PR DESCRIPTION
**Pull request by bapu-nexl**

### CODE REVIEW

File: .byebug_history [hunk 1]
```diff
@@ -0,0 +1,91 @@
+continue
```
Comment (grantspeelman) [inline .byebug_history]: @bapu-nexl you might want to remove this file and add it to .gitignore

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+#
+# See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+require 'byebug'
+require 'securerandom.rb'
+require 'faker'
+require File.dirname(__FILE__) + '/../lib/transferwise_client.rb'
+
+TransferwiseClient.configure do |config|
+  config.profile_id = '3431'
+  config.auth_key = '4a569d2b-264c-46ef-8280-7d8e96ac5b10'
```
Comment (grantspeelman) [inline spec/spec_helper.rb:23]: whoops, you accidentally added this key information into source control. 
Seeing that this is a public repo, these are now compromised.
We should get new keys from transferwise.

**Review by grantspeelman — CHANGES_REQUESTED** [review]
Looks good so far, nice work 👍 

Please remove transferwise keys
  and
Think you can  remove `coverage` folder from git and add it to `.gitignore`

Design looks good though 👍

File: spec/spec_helper.rb (hunk 2 above)
Comment (bapu-nexl) [reply spec/spec_helper.rb:23]: No, thats ok. Its key for sandbox endpoint with my own account. I wanted to write some integration tests. But i think i will not write any integration tests and remove these keys.

File: .byebug_history (hunk 1 above)
Comment (bapu-nexl) [reply .byebug_history]: i will add this file and the coverage files here.

**Comment by bapu-nexl** [comment]
Thanks for the review i wanted to know if these interfaces makes sense.

**Comment by grantspeelman** [comment]
Yes interfaces make sense 👍

**Pull request merged**